motoman_motopos_mh1655_support/urdf/motopos_mh1655_ts_macro.xacro

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:macro name="motoman_motopos_mh1655_ts" params="prefix use_pedestal:=false">
++
+    <xacro:include filename="$(find motoman_resources)/urdf/common_materials.xacro"/>
+    <xacro:property name="pedestal_offset" value="0.695" />
```

Comment:
This file is essentially a duplicate of what we have in the `forge_descriptions` repo. Files in this repo are reserved for stuff we eventually would like to get into `motoman's` main repo.

@Doomerdinger is this stuff that we want in this repository? If so should we remove the files from `forge_descriptions`?

Reply:
Yeah so there's a few things here.
This repo is reserved for things we want to contribute back into the upstream `motoman` repo. This may be one of those things, but we have to be very particular about this stuff. I imagine this tailstock isn't just called `mh1655_ts` in reality. I bet it has its own model number and whatnot.

I am also not sure there is a standard convention for something like `use_pedestal` but default false seems simple enough.

Lastly, this should branch off of the `kinetic-devel` branch, not the `forge` branch. That allows us to make a PR into the upstream. Then we should additionally make a PR from that branch into `forge`. `forge` is our own personal branch for this stuff, with any required modifications that we need for forge, whereas the other branch only contains things that are required for the official upstream motoman repo. In the case of this file I expect them to be identical

Reply:
If we already have this stuff in forge, it is up to us. We can use this and remove it from forge (we'd need to make an adapter in forge if there isn't one already) or we can close this PR and leave it for later, continuing to use what we have in forge.

Reply:
Appreciate the thoroughness @Doomerdinger. @ruslan-path are the collision models different than the ones in forge descriptions? 

I found the yaskawa page for the positioner but do not see the [model number](https://www.motoman.com/en-us/products/peripherals/positioners/headstock-tailstock) listed. Good call on the use pedestal. Digging through the robot arms in the motoman repo it appears they exclusively use the prefix argument and nothing else. We would have to avoid the `use_pedestal` argument to get this upstream.

Personally I would prefer to keep what we have in forge and close this PR

Reply:
The collision models are the same. The reason I moving them is to align TS with the HS and have same configuration hierarchy.
When you say close the PR, do you mean not fix the issue of TS visual not being available in the GUI preview or fix it another way?

Reply:
I guess I can just move mh1655 visuals under generic, that should satisfy gui. I still would like to see better/more readable structure of this files instead of pile of generic containers in the future.

Reply:
So since the consensus is to kill this PR, I'll update forge PR and will be creating new GUI PR to move files under generic folder.